runners/katacoda/templates/dockerCompose.md

```diff
@@ -0,0 +1,21 @@
+<%= text; %>
+
+## Docker Compose
```

Comment:
This line can be removed. Katacoda displays the text from the index.json as the title. So it would be displayed twice